### ISSUE TITLE
renderfunction not working in v1.2.4

### ISSUE THREAD

**Issue by aleex1848** [jbaysolutions/vue2-bootstrap-table#35]
Hi,

i'm using renderfunction a lot to display buttons.
after updating from v1.2.1 to v1.2.4 the renderfunctions stop working. there is simply nothing rendered.
Downgrading back to v1.2.1 solves the problem.
Example:

table definition:
```
{
    title: "Aktionen",
    renderfunction: this.renderAktionen,
},  
```

renderfunction:
```
renderAktionen: function(name, entry) {            
            let info = '<button data-typ="info" data-target="'+entry.id+'" class="btn btn-xs btn-primary">Info</button>';            
            if(entry.status == 'validiert')
            {
                var loeschen = '<button data-typ="ablehnen" data-target="'+entry.id+'" class="btn btn-xs btn-danger">Ablehnen</button>';            
            }
            else
            {
                var loeschen = ''
            }
            return info + loeschen;
            
        },
```

Output v1.2.1
![image](https://user-images.githubusercontent.com/17452861/161995288-c065315a-296f-4ea7-afe2-a60ff15c62b0.png)

Output v1.2.4
![image](https://user-images.githubusercontent.com/17452861/161995381-bab54d63-f12e-4221-87cd-73e4d1b3085f.png)

**Comment by gmsa** [comment]
Which version of Vue are you using?

**Comment by aleex1848** [comment]
> 

I'm using Vue 2.6.14 on this app.

**Comment by pepperbilly** [comment]
Same over here. No Warnings or Console Output, the Renderfunctions will not be called at all.
I'm using Vue 2.6.14.

**Comment by syshex** [comment]
Hey, can you provide a jsfiddle of this case ?

**Comment by gmsa** [comment]
fixed on release 1.2.5
